### PR TITLE
Add to calendar feature added

### PR DESCRIPTION
**Pull request by chhavip**
Clicking on time will now lead user to add the event to their calendar, with most of the options pre-filled. With reference to #118

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java [hunk 1]
```diff
@@ -78,6 +81,20 @@ public void onCreate(Bundle savedInstanceState) {
         } else {
             timings = start + " - " + end;
             text_time.setText(timings);
+            text_time.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    Intent intent = new Intent(Intent.ACTION_INSERT);
+                    intent.setType("vnd.android.cursor.item/event");
+                    intent.putExtra(CalendarContract.Events.TITLE, title);
```
Comment (championswimmer) [inline app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java]: variable `title` should be declared final, or else it wont build

**Comment by chhavip** [comment]
Similar constraint with any other variable?

**Comment by championswimmer** [comment]
Nope just this.

**Comment by championswimmer** [comment]
You **might** need to set variable `session` final too. Just make sure it builds before you update the PR :)

**Comment by chhavip** [comment]
I have updated the PR, please review.

**Pull request merged**